### ISSUE TITLE
Deletion in the overview page not working properly

### ISSUE THREAD

**Issue by siv2r** [siv2r/kidney-exchange#45]
I can see pd-pair details on the summary page, but those are not displayed on the overview page (where deletion of a pair is possible). 

**Note:** If the user encounters #35 during insertion. Then patient files and donor files will not be inserted into the database whereas other details will be inserted. Maybe no insertion happens in the pd_pairs table during this error?? :thinking:

**Comment by Sukriti-sood** [comment]
@siv2r  Is this issue Resolved..............?

**Comment by siv2r** [comment]
@Sukriti-sood I don't think so, since #35 is not solved yet and this is the result of that bug.

**Comment by yash120102** [comment]
@siv2r Please let me know if issue #35 is solved. I can see a PR in that. If that is resolved then can i take this issue?

**Comment by siv2r** [comment]
@yash120102 #35 is not resolved. You can try working on that

**Comment by HarshKumarChoudary** [comment]
I'm GSSOC 22 contributor. If this issue is still available, please assign it to me. Also please add GSSOC22 label for that. Thanks

**Comment by purohitmansi** [comment]
Hello @siv2r!

I am a GSSoC22 contributor. I would love to work on this issue and make PR as soon as I resolve the issue. Please do assign this to me.

ThankYou:)
